readme.md

```diff
@@ -124,7 +140,89 @@ There is no default export.
 
 Default schema ([`Schema`][api-schema]).
 
-Follows [GitHub][] style sanitation.
+Follows [GitHub][] style sanitation. 
+
+#### Overriding the default schema
```

Comment:
This should be in a separate `Example` section, see for example https://github.com/rehypejs/rehype-sanitize?tab=readme-ov-file#example